server/data/data-jmap/src/main/java/org/apache/james/jmap/api/projections/MessageFastViewProjectionHealthCheck.java

```diff
@@ -53,23 +53,23 @@ public ComponentName componentName() {
 
     @Override
     public Mono<Result> check() {
-        return Mono.fromCallable(() -> retrieveMissCountMetric.getCount())
+        return Mono.fromCallable(retrieveMissCountMetric::movingAverage)
             .flatMap(missCount -> {
                 if (missCount == 0) {
                     return Mono.just(Result.healthy(COMPONENT_NAME));
                 } else {
-                    return Mono.fromCallable(() -> retrieveHitCountMetric.getCount())
+                    return Mono.fromCallable(retrieveHitCountMetric::movingAverage)
                         .map(hitCount -> check(hitCount, missCount));
                 }
             });
     }
 
-    private Result check(long hitCount, long missCount) {
-        long totalCount = hitCount + missCount;
+    private Result check(double hitCount, double missCount) {
```

Comment:
please explain what is a count in `double`?

Reply:
We should call this 'average'